src/packages/@ncigdc/routes/UserProfileRoute/UserProfileRoute.js

```diff
@@ -0,0 +1,66 @@
+/* @flow */
+import React from 'react';
+import { compose, branch, renderComponent } from 'recompose';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router';
+
+import { Row } from '@ncigdc/uikit/Flex';
+import Hidden from '@ncigdc/components/Hidden';
+import Heading from '@ncigdc/uikit/Heading';
+import Table, { Tr, Td } from '@ncigdc/uikit/Table';
+import CheckedSquareO from '@ncigdc/theme/icons/CheckedSquareO';
```

Comment:
How would you feel about just a checkmark without the square, and nothing if they don't have it? These make me feel like I can change my permissions haha

Reply:
haha true